mmdet3d/datasets/pipelines/test_time_aug.py

```diff
@@ -77,11 +78,11 @@ def __call__(self, results):
         # modified from `flip_aug = [False, True] if self.flip else [False]`
         # to reduce unnecessary scenes when using double flip augmentation
         # during test time
-        flip_aug = [True] if self.flip else [False]
+        flip_aug = [False, True] if self.flip else [False]
```

Comment:
This seems to be a conflict. Please make sure this modification will not affect current models.